src/Components/Fsi.fs

```diff
@@ -425,18 +426,23 @@ module Fsi =
             |> ignore)
 
     let private handleCloseTerminal (terminal : Terminal) =
-        fsiOutputPID
```

Comment:
`handleCloseTerminal` needs to reset the state. Otherwise, we are broken for following scenario:
1. Close terminal
2. Run `FSI: Send Selection`

Reply:
theres a bit of a race condition with closing it out during startup then, I'd imagine